terraform/etl/49-lambda-streetscene-ingestion.tf

```diff
@@ -146,10 +146,10 @@ module "street-systems-api-ingestion" {
     API_URL = "https://flask-customer-api.ki8kabg62o4fg.eu-west-2.cs.amazonlightsail.com"
   }
   layers = [
-    "arn:aws:lambda:eu-west-2:120038763019:layer:requests-2-31-0-and-httplib-0-22-0-layer:1",
-    "arn:aws:lambda:eu-west-2:336392948345:layer:AWSSDKPandas-Python39:12",
-    "arn:aws:lambda:eu-west-2:120038763019:layer:s3fs-2023-12-2-layer:1",
-    "arn:aws:lambda:eu-west-2:120038763019:layer:urllib3-1-26-18-layer:1"
+    "arn:aws:lambda:eu-west-2:${data.aws_caller_identity.data_platform.account_id}:layer:requests-2-31-0-and-httplib-0-22-0-layer:1",
+    "arn:aws:lambda:eu-west-2:${data.aws_caller_identity.data_platform.account_id}:layer:AWSSDKPandas-Python39:12",
```

Comment:
This one doesn't need changing, it's one of the AWS layers and isn't in our account